src/benchmark.cc

```diff
@@ -67,7 +67,11 @@ DEFINE_string(benchmark_format, "tabular",
               "The format to use for console output. Valid values are "
               "'tabular', 'json', or 'csv'.");
 
-DEFINE_bool(color_print, true, "Enables colorized logging.");
+DEFINE_string(color_print, "auto",
```

Comment:
a boolean might be better with default 'true' being overridden if the TERM doesn't support it.